packages/maskbook/src/plugins/RedPacket/database.ts

```diff
@@ -15,7 +15,10 @@ export async function getRedPacket(rpid: string) {
     return record ? RedPacketRecordOutDB(record) : undefined
 }
 
-export function addRedPacket(record: RedPacketRecord) {
+export async function addRedPacket(record: RedPacketRecord) {
+    if (!isNil(await RedPacketDatabase.get('red-packet', record.id))) {
```

Comment:
Why you need isNil?

Reply:
It's return type is `RedPacketRecord | undefined`. It's unnecessary to use lodash